Client/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -642,24 +642,52 @@ extension BrowserViewController: UIContextMenuInteractionDelegate {
     return UIContextMenuConfiguration(identifier: nil, previewProvider: nil) { [unowned self] _ in
       var actionMenuChildren: [UIAction] = []
 
-      let pasteGoAction = UIAction(
-        title: Strings.pasteAndGoTitle,
-        image: UIImage(systemName: "doc.on.clipboard.fill"),
-        handler: UIAction.deferredActionHandler { _ in
-          if let pasteboardContents = UIPasteboard.general.string {
-            self.topToolbar(self.topToolbar, didSubmitText: pasteboardContents)
-          }
-        })
+      var pasteGoAction: UIAction?
+      var pasteAction: UIAction?
+      
+      if #available(iOS 15.0, *) {
+        pasteGoAction = UIAction(
+          title: Strings.pasteAndGoTitle,
+          image: UIImage(systemName: "doc.on.clipboard.fill"),
+          identifier: .pasteAndGo,
+          handler: UIAction.deferredActionHandler { _ in
+            if let pasteboardContents = UIPasteboard.general.string {
+              self.topToolbar(self.topToolbar, didSubmitText: pasteboardContents)
+            }
+          })
+      } else {
+        pasteGoAction = UIAction(
+          title: Strings.pasteAndGoTitle,
+          image: UIImage(systemName: "doc.on.clipboard.fill"),
+          handler: UIAction.deferredActionHandler { _ in
+            if let pasteboardContents = UIPasteboard.general.string {
+              self.topToolbar(self.topToolbar, didSubmitText: pasteboardContents)
+            }
+          })
+      }
```

Comment:
Why not just setup the identifier in the availability check instead of the whole action? Or even make a back ported definition for the identifiers so we can reuse them wherever needed

i.e.

```swift
extension UIAction.Identifier {
  static var backportedPasteAndGo: UIAction.Identifier? {
    if #unavailable(iOS 15.0) {
      return nil
    }
    return .pasteAndGo
  }
  static var backportedPaste: UIAction.Identifier? {
    if #unavailable(iOS 15.0) {
      return nil
    }
    return .paste
  }
}

...

UIAction(
  title: Strings.pasteAndGoTitle,
  image: UIImage(systemName: "doc.on.clipboard.fill"),
  identifier: .backportedPasteAndGo,
  handler: UIAction.deferredActionHandler { _ in
    if let pasteboardContents = UIPasteboard.general.string {
      self.topToolbar(self.topToolbar, didSubmitText: pasteboardContents)
    }
  }
)
```

Reply:
Sounds great, changing it.

Reply:
Just realizing that code above probably doesn't compile and the `unavailable` checks will need to be replaced with `#available` and swapping the nil/paste :)